### DISCUSSION TITLE
Invalid cookie when using yahooquery 2.3.3

### DISCUSSION THREAD

**Discussion by yafm** [dpguthrie/yahooquery#242] [Q&A]
>>> from yahooquery import Ticker
>>> aapl = Ticker('aapl')
>>> aapl.summary_detail
{'aapl': 'Invalid Cookie'}

Something seems to be broken

**Comment by srujantaticherla** [comment]
I also get a similar issue when I am using the premium version 

ticker = Ticker(ticker_symbol, driver=driver,username='xxxx@yahoo.com', password='xxx')

AttributeError: 'YahooFinanceHeadless' object has no attribute 'cookies'. I am running the code from sage maker. But the code works fine from local pycharm

**Comment by yafm** [comment]
It works with 2.3.7, so closing
